src/security-council-mgmt/governors/SecurityCouncilNomineeElectionGovernor.sol

```diff
@@ -208,23 +210,27 @@ contract SecurityCouncilNomineeElectionGovernor is
         }
     }
 
-    /// @notice Put `msg.sender` up for nomination. Must be called before a contender can receive votes.
-    ///         Contenders are expected to control an address than can create a signature that would be a
-    ///         recognised by a Gnosis Safe. They need to be able to do this with this same address on each of the
-    ///         chains where the Security Council is active. It is expected that the nominee vetter will check this
-    ///         during the vetting phase and exclude any contenders which dont meet this criteria.
-    /// @dev    Can be called only while a proposal is active (in voting phase)
+    /// @notice Put `contender` up for nomination. Must be called before a contender can receive votes.
+    /// @param  proposalId The id of the proposal
+    /// @param  contender The address of the contender
+    /// @param  signature EIP712 `AddContenderMessage(uint256 proposalId,address contender)` signed by `contender`
+    /// @dev    Can be called only while a proposal is pending (after proposal created but before voting phase)
     ///         A contender cannot be a member of the opposite cohort.
-    function addContender(uint256 proposalId) external {
+    function addContender(uint256 proposalId, address contender, bytes calldata signature) external {
```

Comment:
I don't think there is a big risk but allowing contender to be different from msg.sender allow someone to just spam contender with a single helper contract

Reply:
yeah i don't think that's much worse than what's currently possible (contract spawns many new contracts to add themselves in a single tx)

Reply:
i think it'll be spammable no matter what we do